index.html

```diff
@@ -54,133 +61,230 @@ <h3 id="connect"><a href="#">LET'S CONNECT</a></h3>
             <li class="big works">
-                <img src="assets/SnapshootPortfolio4.png" alt="Snapshoot">
-                <h1>Multi-Post<br>Stories</h1>
-                <ul>
-                    <li>CANOPY</li>
-                    <li class="qualifications"><img src="assets/Counter.png" alt="Counter">Back-end dev</li>
-                    <li class="qualifications"><img src="assets/Counter.png" alt="Counter">2015</li>
-                </ul>
-                <p>
-                    A daily selection of privately personalized reads; no accounts or sign-ups required.
-                </p>
-
-                <ul class="languages">
-                    <li>html</li>
-                    <li>css</li>
-                    <li>javaScript</li>
-                </ul>
-                <h2><a href="#">See Project</a></h2>
+                <img class="mobile-img" src="assets/SnapshootPortfolio4.png" alt="Snapshoot">                
+                <div>
+                    <h1>Multi-Post<br>Stories</h1>
+                    <ul>
+                        <li>CANOPY</li>
+                        <li class="qualifications"><img src="assets/Counter.png" alt="Counter">Back-end dev</li>
+                        <li class="qualifications"><img src="assets/Counter.png" alt="Counter">2015</li>
+                    </ul>
+                    <p>
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                    <ul class="languages">
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javaScript</li>
+                    </ul>
+                    <h2><a href="#">See Project</a></h2>
+                </div>
             </li>
         </ul>
     </section>
-    <section class="contact">
-        <h1>About<br>Myself</h1>
-        <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my
-            work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact
-            me.
-        </p>
-        <h3 id="connect2"><a href="#">LET'S CONNECT</a></h3>
-        <ul class="social">
-            <li><a href="#"><img src="assets/Git.png" alt="Twitter"></a></li>
-            <li><a href="#"><img src="assets/Linkedin.png" alt="Linkedin"></a></li>
-            <li><a href="#"><img src="assets/Hand.png" alt="M"></a></li>
-            <li><a href="#"><img src="assets/Twitter.png" alt="GitHub"></a></li>
-            <li><a href="#"><img src="assets/M.png" alt="Hand"></a></li>
-        </ul>
-        <h2><a href="#">Get my resume</a></h2>
-        <ul id="about-lenguages">
-            <li>
-                <div class="section-title">
-                    <h3>Lenguages</h3><img src="assets/UnionDown.png" alt="arrowdown">
+    <section class="container-works-desktop">
+        <ul>
+            <li class="works">                
```

Comment:
- [ ] Your design is very close to following the provided design in [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A418). However, your design appears overly stretched on smaller screens and smaller on larger screens. Due to this, your page overflows, and therefore the full width (`100%`) of the page does not show certain parts of the screen, and hence a horizontal scrollbar displays beneath the window of the page. 

Your design:
![shot3](https://user-images.githubusercontent.com/12745474/121730393-89051580-cae7-11eb-9dc0-53fc01f3d28a.jpeg)

Your `<li>` tags bearing the class name `works` are shifted outside the main page wrapper:
![shot4](https://user-images.githubusercontent.com/12745474/121731664-2876d800-cae9-11eb-9946-66b7e94ffdb0.jpeg)

Commenting out line  `574` of your `style/style.css` : `width: 1156px;` solves  95% of the issue. Divs and elements which stretches across the screen should have a width of `100%`

Let's ensure to correct the above

Reply:
Thank you for the review! 

If I take out the width in line 574,  the content is going to bleed out of the container since neither the font or the image size is being altered. Would you suggest that I change the image size to % or the div with the text should be changed? I'm at a loss and need some help.

I had the idea of setting the size as a fixed value since it was not asked and I didn't want to mess the content, so I did not think that the horizontal scroll bar would be a problem, since my coding partner did this way and it got accepted. 

Best regards,

Guilherme

Reply:
Mobile:
![image](https://user-images.githubusercontent.com/81584449/121755369-5b22cf80-caed-11eb-9ef4-e389c838b277.png)

Dektop Croped to 1092px:
![image](https://user-images.githubusercontent.com/81584449/121755495-b785ef00-caed-11eb-8efc-b13c188a5df0.png)

Desktop Croped to 1200px:
![image](https://user-images.githubusercontent.com/81584449/121755590-f61ba980-caed-11eb-9795-091abf60b24e.png)

Reply:
Hello Guilherme,

Congrats on completing this project! :clap: :clap: :clap:

One of the rubrics of this project was to ensure that we understand the best practices for a good layout, which we would likely implement in our front-end project in the near future or so. We were therefore to ensure that our designs match the design of the provided Figma template as much as possible.

I do apologize if there was something I missed during the review.

Cheers, and Happy coding! 🍷 💻

Reply:
Thank you! I will keep that in mind!

Cheers.